src/index.html

```diff
@@ -1,14 +1,42 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta name="viewport" content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
+  <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
+
 </head>
+
 <body>
-  <h1>Moyo header</h1>
+  <header class="main">
+    <div class="logo">
+      <img src="images/logo.png" alt="logo moyo">
+    </div>
+
+    <nav class="navigation">
+      <ul class="list">
+        <li class="list__elem"> <a href="#" class="list__link is-active">apple</a> </li>
```

Comment:
не пиши все в один рядок
```suggestion
        <li class="list__elem">
          <a href="#" class="list__link is-active">apple</a>
        </li>
```